reana_workflow_controller/rest/workflows.py

```diff
@@ -254,7 +255,9 @@ def get_workflows(paginate=None):  # noqa
             workflow_status = [RunStatus[status] for status in status_list.split(",")]
             query = query.filter(Workflow.status.in_(workflow_status))
         if workflow:
```

Comment:
The `workflow` variable name here might be a bit confusing as it looks like an instance of the Workflow model, WDYT about keeping `workflow_id_or_name`?

IMHO, we should use marshmallow schemas [as we do for r-server](https://github.com/reanahub/reana-server/blob/b6cd672a94975124221da4875c17ebe50dd82ae5/reana_server/rest/workflows.py#L61) to avoid all the manual variable assignments of request args. It doesn't belong to this task though.

Reply:
Indeed `workflow` variable name was confusing here, changed it to `workflow_id_or_name`.
Regarding the `marshmallow schemas`, totally agree! It would be nice to unify and use it in all the endpoints, perhaps worth to create an issue for that?

Reply:
> Regarding the `marshmallow schemas`, totally agree! It would be nice to unify and use it in all the endpoints, perhaps worth to create an issue for that?

I wouldn't mind at all! However, we probably should [upgrade to Marshmallow 3](https://github.com/reanahub/reana-job-controller/issues/352) first.